docs/src/test/scala/docs/stream/cookbook/RecipeAdhocSource.scala

```diff
@@ -13,12 +13,14 @@
 
 package docs.stream.cookbook
 
-import java.util.concurrent.atomic.{ AtomicBoolean, AtomicInteger }
 
-import org.apache.pekko.stream.scaladsl.Source
-import org.apache.pekko.stream.testkit.scaladsl.TestSink
-import org.apache.pekko.testkit.TimingTest
-import org.apache.pekko.{ Done, NotUsed }
+import java.util.concurrent.atomic.{AtomicBoolean, AtomicInteger}
+import org.apache.pekko
+import pekko.stream.scaladsl.Source
+import pekko.stream.testkit.scaladsl.TestSink
+import pekko.stream.BackpressureTimeoutException
+import pekko.testkit.TimingTest
+import pekko.{Done, NotUsed}
```

Comment:
looks like a scalafrmt issue will happen - should be `pekko.{ Done, NotUsed }`  - also line 17